scripts/conversion_script.py

```diff
@@ -81,6 +81,55 @@ def to_dict(self):
         }
 
 
+def _check_ini_file_modified():
+    rpm_va_output, ini_file_not_modified = run_subprocess(
+        ["/usr/bin/rpm", "-Va", "convert2rhel"]
+    )
+
+    # No modifications at all
+    if not ini_file_not_modified:
+        return False
+
+    lines = rpm_va_output.strip().split("\n")
+    for line in lines:
+        line = line.strip().split()
+        status = line[0].replace(".", "").replace("?", "")
+        path = line[-1]
+
+        default_ini_modified = path == "/etc/convert2rhel.ini"
+        md5_hash_mismatch = "5" in status
+
+        if default_ini_modified and md5_hash_mismatch:
+            return True
+    return False
+
+
+def check_convert2rhel_inhibitors_before_run():
+    """
+    Conditions that must be True in order to run convert2rhel command.
+    """
+    ini_report = (
+        "Either remove the %s file by running "
+        "'rm -f %s' or uninstall convert2rhel by running "
+        "'yum remove convert2rhel' before running the Task again."
+    )
+    default_ini_path = "/etc/convert2rhel.ini"
+    custom_ini_path = os.path.expanduser("~/.convert2rhel.ini")
+
+    if os.path.exists(custom_ini_path):
+        raise ProcessError(
+            message="Custom %s was found." % custom_ini_path,
+            report=ini_report % (custom_ini_path, custom_ini_path),
```

Comment:
Ah, now I see that you use the same message even for the ~/.convert2rhel.ini file. To resolve this case, the customer needs to remove the ini file. Uninstalling convert2rhel won't help because the custom ini file is not owned by the package. So we need to have different report messages for /etc/convert2rhel.ini and ~/.convert2rhel.ini.